contrib/Multi-Stat/index.php

```diff
@@ -0,0 +1,145 @@
+<?php require_once("settings.php"); ?>
+<html>
+<head>
+<title><?php echo $siteTitle; ?></title>
+<link rel="stylesheet" href="css/style.css" />
+<link rel="icon" type="image/png" href="https://kaosincorporated.com/assets/images/logo.png">
+<script src="https://kit.fontawesome.com/97c7e13229.js" crossorigin="anonymous"></script>
+       <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="css/bootstrap.css">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="css/bootstrap-theme.min.css">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="js/bootstrap.min.js"></script>
+	
+</head>
+<body>
+<center>
+<div class="container">
+	<img src="images/logo.png" class="logo" />
+	<div class="header"><?php echo $siteName; ?></div>
+		<div class="list pull-left">
+		<?php
+		//Host information
+		$serverIP = $javaIP;
+
+		foreach ($serverIP as $ip)
+		{
+			$urlParts = explode('.', $ip);
+
+			$port = "25565"; // Port for the server to listen on, bedrock is 19132 by default, Java is 25565
+
+			// Valid Protocols: Unknown, Beta, Legacy, ExtendedLegacy, Json, BedrockRaknet
+			// Use BedrockRakNet for Bedrock/Pocket Edition, The rest will not work with bedrock servers.  Also, BedrockRaknet does not output player usernames.
+			// ExtendedLegacy does not seem to work.  This is a Minestat issue or a Server problem.  
+			// For Java, use Json or Legacy, Json seems to be more accurate at locating online servers vs Legacy does.  Most likely because the servers are not using Geyser/Floodgate?
+			
+			$protocol = "Json";
+
+
+			$runCMD = "MineStat -Address ".$ip." -Port ".$port." -Protocol ".$protocol." -Timeout 1";
+			$output = shell_exec('powershell.exe '.$runCMD);
+			$output = str_replace('}', ', ', str_replace('{.', '', $output)); //removes brackets and dots from usernames
+			if(str_contains($output, "Offline"))
+			{
+				 $javaStatus = false; // Return Java server status
+				 $statusType = "Offline";
+			}
+			elseif(str_contains($output, "Timeout"))
+			{
+				$javaStatus = false; // Return Java server status
+				$statusType = "Timed Out";
+			}
+			elseif(str_contains($output, "ConnFail"))
+			{
+				$javaStatus = false; // Return Java server status
+				$statusType = "Connection Failure";
+			}
+			else
+			{
+				$javaStatus = true;
+			}
+			if($javaStatus != true)
+			{
+					echo '<div class="alert alert-danger">Java Server: '.$ip.':'.$port.' is currently Offline!<br />Status Type: '.$statusType.'</div>';
+			}
+			else
+			{
+				if(isset($urlParts[1]))
+				{
+					echo "<pre class='item text-left'><center>".ucfirst($urlParts[1])." - Java</center>".$output."</pre><br />";
+				}
+				else
+				{
+					echo "<pre class='item text-left'><center>".ucfirst($urlParts[0])." - Java</center>".$output."</pre><br />";
+				}
+			}
+		}

```

Comment:
```suggestion
		foreach ($serverIP as $ip)
		{
			$urlParts = explode('.', $ip);

			$port = "25565"; // Port for the server to listen on, bedrock is 19132 by default, Java is 25565

			// Valid Protocols: Unknown, Beta, Legacy, ExtendedLegacy, Json, BedrockRaknet
			// Use BedrockRakNet for Bedrock/Pocket Edition, The rest will not work with bedrock servers.  Also, BedrockRaknet does not output player usernames.
			// ExtendedLegacy does not seem to work.  This is a Minestat issue or a Server problem.  
			// For Java, use Json or Legacy, Json seems to be more accurate at locating online servers vs Legacy does.  Most likely because the servers are not using Geyser/Floodgate?
			
			$protocol = "Json";


			$runCMD = "MineStat -Address ".$ip." -Port ".$port." -Protocol ".$protocol." -Timeout 1";
			$output = shell_exec('powershell.exe '.$runCMD);
			$output = str_replace('}', ', ', str_replace('{.', '', $output)); //removes brackets and dots from usernames
			if(str_contains($output, "Offline"))
			{
				 $javaStatus = false; // Return Java server status
				 $statusType = "Offline";
			}
			elseif(str_contains($output, "Timeout"))
			{
				$javaStatus = false; // Return Java server status
				$statusType = "Timed Out";
			}
			elseif(str_contains($output, "ConnFail"))
			{
				$javaStatus = false; // Return Java server status
				$statusType = "Connection Failure";
			}
			else
			{
				$javaStatus = true;
			}
			if($javaStatus != true)
			{
					echo '<div class="alert alert-danger">Java Server: '.$ip.':'.$port.' is currently Offline!<br />Status Type: '.$statusType.'</div>';
			}
			else
			{
				if(isset($urlParts[1]))
				{
					echo "<pre class='item text-left'><center>".ucfirst($urlParts[1])." - Java</center>".$output."</pre><br />";
				}
				else
				{
					echo "<pre class='item text-left'><center>".ucfirst($urlParts[0])." - Java</center>".$output."</pre><br />";
				}
			}
		}
foreach ($serverIP as $ip)
		{
      $urlParts = explode('.', $ip);
      end($urlParts);
			$urlParts = prev($urlParts);

			$port = "25565"; // Port for the server to listen on, bedrock is 19132 by default, Java is 25565

			// Valid Protocols: Unknown, Beta, Legacy, ExtendedLegacy, Json, BedrockRaknet, Query
			// Use BedrockRakNet for Bedrock/Pocket Edition, The rest will not work with bedrock servers.  Also, BedrockRaknet does not output player usernames.
			// ExtendedLegacy does not seem to work.  This is a Minestat issue or a Server problem.  
			// For Java, use Json or Legacy, Json seems to be more accurate at locating online servers vs Legacy does.  Most likely because the servers are not using Geyser/Floodgate?
      $protocol = "Json";

			$runCMD = '"(MineStat -Address '.$ip.' -Port '.$port.' -Protocol '.$protocol.' -Timeout 1 | ConvertTo-Json)"';
      $output = json_decode(mb_convert_encoding(shell_exec('PowerShell.exe '.$runCMD), 'UTF-8', 'UTF-8'), true);

      // Check for JSON decoding errors
      if (json_last_error() !== JSON_ERROR_NONE) {
        // Handle the error here, you can log it or return a custom error message
      }

      $javaStatus = false;
      $statusType = "Offline";
      switch ($output["connection_status"]) {
        case 'Success':
			    $statusType = "Online";
          $javaStatus = true;
          break;
        case 'Connfail':
			    $statusType = "Connection Failure";
          break;
        case 'Timeout':
			    $statusType = "Timed Out";
          break;
      }

      $ip = $output["address"];
      $port = $output["port"];
      $version = $output["version"];
      $motd = $output["stripped_motd"];
      $current_players = $output["current_players"];
      $max_players = $output["max_players"];
      $latency = $output["latency"];
      //more available: https://github.com/FragLand/minestat/tree/master/PowerShell
      $output = "\n\naddress         : $ip\n"
              . "port            : $port\n"
              . "version         : $version\n"
              . "motd            : $motd\n"
              . "current_players : $current_players\n"
              . "max_players     : $max_players\n"
              . "latency         : $latency\n\n\n";
      
			if($javaStatus != true)
				echo '<div class="alert alert-danger">Java Server: '.$ip.':'.$port.' is currently Offline!<br />Status Type: '.$statusType.'</div>';
			else
				echo "<pre class='item text-left'><center>$urlParts - $protocol</center>".$output."</pre><br />";
		}
```